internal/controller/ops/watched/reconciler_test.go

```diff
@@ -935,94 +935,6 @@ func TestNewOperation(t *testing.T) {
 				},
 			},
 		},
-		"SyntheticDeletedResource": {
```

Comment:
Deleted because this is testing a codepath downstream of naming the operation, so it does not test anything that the above tests test.

Reply:
i think this test case is still reasonable to keep - it's making sure that `NewOperation()` honors the special synthetic deleted resource version and that value makes it into the annotations of the created `Operation` object.  None of the preceding test cases have a non-empty resource version that gets set as an annotation on the `Operation` object, so this test case does seem to have at least a bit of unique coverage from that perspective. I think we should keep it - what do you think about that?